gulp buildでlint error

`.env`は`.env.sample`と同じ内容

```
➜  float-behind-client git:(34bb95d) gulp build
[23:13:11] Requiring external module babel-register
[23:13:11] Warning: gulp version mismatch:
[23:13:11] Global gulp is 3.9.1
[23:13:11] Local gulp is 3.9.0
[23:13:12] Using gulpfile ~/repo/float-behind-client/gulpfile.babel.js
[23:13:12] Starting 'lint'...
[23:13:12] Starting 'webpack'...
[23:13:13] Starting 'styles'...
[23:13:13] Starting 'images'...
[23:13:13] Starting 'fonts'...
[23:13:13] Finished 'images' after 236 ms
[23:13:13] Finished 'fonts' after 27 ms
[23:13:17] Finished 'styles' after 4.11 s
[23:13:17] Starting 'inject'...
[23:13:18] gulp-inject 3 files into index.html.
[23:13:18] Finished 'inject' after 974 ms
[23:13:19] Finished 'webpack' after 6.84 s
[23:13:19] Starting 'html'...
[23:13:20] Finished 'html' after 356 ms
[23:13:27]
app/utils/ga.js
  3:14  error  "process" is not defined  no-undef
  3:33  error  "process" is not defined  no-undef

✖ 2 problems (2 errors, 0 warnings)

[23:13:27] 'lint' errored after 15 s
[23:13:27] ESLintError in plugin 'gulp-eslint'
Message:
    Failed with 2 errors
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -6,6 +6,7 @@
     node: false
   globals:
     Native: true
+    process: true
   ecmaFeatures:
     arrowFunctions: true
     blockBindings: true

